mycroft/gui/__init__.py

```diff
@@ -37,7 +37,8 @@ def __init__(self, skill):
 
     @property
     def bus(self):
-        return self.skill.bus
+        if self.skill:
```

Comment:
Should this log an error or init a client the else case?

Reply:
this should not be done in this PR, accidental commit, i ran into this because of other issue and commited while debugging, its something we should look at later on but unrelated to this PR